apps/teamsjs-diagnostic-app/src/apis/AppApi.tsx

```diff
@@ -0,0 +1,25 @@
+import { app, ResumeContext } from '@microsoft/teams-js';
+
+export const registerOnResume = async (): Promise<string> => {
+    console.log("Calling registerOnResume API");
```

Comment:
Is this console.log still needed or was this used for testing?

Reply:
It is needed! There are lots of console.log everywhere because that is how I am doing the diagnostic logging - I have a file called LoggerUtility where I essentially override the console.log so it appears on my log